README.md

```diff
@@ -42,6 +42,8 @@ For Items, the fields are placed in the `properties`. For Collections, the field
 | sci:doi          | string               | The DOI of the data, e.g. `10.1000/xyz123`. This MUST NOT be a DOIs link. For all DOI names respective DOI links SHOULD be added to the links section (see chapter "Relation types"). |
 | sci:citation     | string               | The recommended human-readable reference (citation) to be used by publications citing the data. No specific citation style is suggested, but the citation should contain all information required to find the publication distinctively. |
 | sci:publications | [[Publication Object](#publication-object)] | List of relevant publications referencing and describing the data. |
+| sci:orcids        | \[string]            | An array of Open Researcher Contribution IDs ([ORCID](https://orcid.org)) associated with this product. For all ORCIDs a link SHOULD be added to the links section.     |
+| sci:ror s         | \[string]            | An array of Research Organization Record ([ROR](https://ror.org)) entity name or unique identifier. For all ROR(s) names and identifiers a link SHOULD be added to the links section. |
```

Comment:
```suggestion
| sci:rors          | \[string]            | An array of Research Organization Record ([ROR](https://ror.org)) entity name or unique identifier. For all ROR(s) names and identifiers a link SHOULD be added to the links section. |
```

Reply:
Do ror need to be an array? Isn't there usually one and then you may have one per asset in addition?

Reply:
Would it make sense to give some best practices here?
For example, sci:orcid makes sense in conjunction with the contacts extension and sci:ror seems to make most sense in the providers list, right?

Reply:
Yes. I was not aware of the contacts extension. I'll have to take a look at it. Should we have an example in the `sci` repo showing the usage with the contacts extension or is the intention that each extension is nominally stand alone?